app/controllers/docker_manager/admin_controller.rb

```diff
@@ -16,8 +16,15 @@ def index
       expected_version = Gem::Version.new('2.0.20220128-1817')
       ruby_version = Gem::Version.new(RUBY_VERSION)
       expected_ruby_version = Gem::Version.new('2.7.6')
+      # Force self-hosters to rebuild if running an older version.
```

Comment:
```suggestion

```
We don't use this plugin internally, so I don't think we need to call out that it's 'self-hosters' only